sdcm/cluster.py

```diff
@@ -4616,13 +4616,15 @@ def node_setup(self, node: BaseNode, verbose: bool = False, timeout: int = 3600)
         if self.params.get('logs_transport') == 'ssh':
             node.install_package('python3')
 
-        if node.distro.is_ubuntu:
-            result = node.remoter.sudo("pro status", ignore_status=True)
+        # check fips enabled only if we specify it in sct config (so we can use ubuntu pro in other cases)
+        if node.distro.is_ubuntu and 'fips' in self.params.get('linux_distro_features'):
```

Comment:
The `linux_distro_features` may be `None` and in this case we will get failure in the condition itself.
So, need to make it be safe coded.

Reply:
it's has a default in `test_default.yaml`, which should suffice.

but I've added a check anyhow